src/language/tekster_nn.ts

```diff
@@ -161,15 +161,16 @@ export default {
   'personopplysninger.spm.riktigAdresse': 'Do you live at this address?',
   'personopplysninger.lesmer-åpne.riktigAdresse':
     'If you have a strictly confidential address',
-  'personopplysninger.lesmer-innhold.riktigAdresse':
-    'If you have a strictly confidential address, your address will not appear here. Answer Yes to this question and continue with the digital application.',
   'personopplysninger.alert.riktigAdresse':
-    'You must provide your true address to the National Registry in order to use this application.',
+    'You must have reported a change of address to the National Registry in order to use this application. You do this at skatteetaten.no.',
```

Comment:
```suggestion
    'You must have reported a change of address to the National Registry in order to use this application. You may do this at skatteetaten.no.',
```
Litt småpirk fra min side men synes `You do this` hørtes litt rart ut :P

Reply:
Takk 👍